README.md

```diff
@@ -0,0 +1,41 @@
+[![GoDoc](https://godoc.org/github.com/ooni/probe-engine?status.svg)](https://godoc.org/github.com/ooni/probe-engine) [![Build Status](https://travis-ci.org/ooni/probe-engine.svg?branch=master)](https://travis-ci.org/ooni/probe-engine) [![Coverage Status](https://coveralls.io/repos/github/ooni/probe-engine/badge.svg?branch=master)](https://coveralls.io/github/ooni/probe-engine?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/ooni/probe-engine)](https://goreportcard.com/report/github.com/ooni/probe-engine)
+
+# OONI probe measurement engine
+
+This repository contains OONI probe's [measurement engine](
+https://github.com/ooni/spec/tree/master/probe#engine). That is, the
+piece of software that implements OONI nettests.
+
+## API
+
+You can [browse ooni/probe-engine's API](
+https://godoc.org/github.com/ooni/probe-engine?status.svg)
```

Comment:
Are you sure this is the right link? When I click on it I only see an image

Reply:
No, it's wrong. I'm fixing it.